source/help/getting-started/configuring-notifications.rst

```diff
@@ -92,11 +92,22 @@ being viewed on desktop.
    from **Account Settings** > **Notifications** > **Mobile Push
    Notifications** > **Trigger push notifications when**.
 
+Team Sidebar Notifications
+----------------------------------------
+
+If Mattermost is open in a browser tab the favicon updates to inform you of unread messages and
+mentions. Browser tab notifications are available on Firefox and Chrome.
+
+- Unread messages are denoted by a small dot left of the team icon.
+- Unread mentions are displayed with a mention counter similar to channels and appears on the top right corner of the team icon.
```

Comment:
Propose: `Unread mentions are displayed with a mention counter that appears on the top right corner of the team icon.`